runtime/common/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "common-runtime"
-version = "100.0.3"
+version = "106.0.0"
```

Comment:
what is the reason for the 6 major version bumb? Keeping it in sync with hydradx-runtime and testing-hydradx-runtime?

Reply:
i believe it is just to match hydradx-runtime and testing-runtime versions. 

but it is necessary ? we dont do this in basilisk and common does not change everytime a runtime changes.